redi-connect-front/src/pages/front/signup/steps/Step4ContactData.tsx

```diff
@@ -124,6 +129,24 @@ export const Step4ContactData = (
           )
         }}
       />
+      <TextField
+        id="githubProfileUrl"
+        name="githubProfileUrl"
+        helperText={touched.githubProfileUrl ? errors.githubProfileUrl : ""}
+        error={touched.githubProfileUrl && Boolean(errors.githubProfileUrl)}
+        label="LinkedIn Profile"
```

Comment:
Hi @typeoh, great work, everything looks good. If you can just fix this label we'll get this merged right away.

Thanks!

Reply:
Thank you for catching that copy pasta error 🙈 will do!